### ISSUE TITLE
Theming Account Creation & Login Block

### ISSUE THREAD

**Issue by willykaram** [NYC-Camp/nyc-camp-2015#56]
Ideally once this block is in place, we can theme it in a similar manner to the BADCamp site, with a nice slider/drop down http://2012.badcamp.net or http://2013.badcamp.net

**Comment by willykaram** [comment]
I've moved this to the registration milestone (next week) as it's not needed for initial site launch.
